facade/src/main/java/org/jboss/pnc/facade/rsql/mapper/DeliverableArtifactRSQLMapper.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2022 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.facade.rsql.mapper;
+
+import org.jboss.pnc.model.DeliverableArtifact;
+import org.jboss.pnc.model.DeliverableArtifactPK;
+import org.jboss.pnc.model.DeliverableArtifact_;
+import org.jboss.pnc.model.GenericEntity;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.persistence.metamodel.SetAttribute;
+import javax.persistence.metamodel.SingularAttribute;
+
+@ApplicationScoped
+public class DeliverableArtifactRSQLMapper extends AbstractRSQLMapper<DeliverableArtifactPK, DeliverableArtifact> {
+
+    public DeliverableArtifactRSQLMapper() {
+        super(DeliverableArtifact.class);
+    }
+
+    @Override
+    protected SingularAttribute<? super DeliverableArtifact, ? extends GenericEntity<?>> toEntity(String name) {
+        switch (name) {
+            case "report":
+                return DeliverableArtifact_.report;
```

Comment:
There is no field `report` in AnalyzedArtifact.
```suggestion
```

Reply:
[2eac15e](https://github.com/project-ncl/pnc/pull/3816/commits/2eac15e287a060a7fc52b038ba3833bd3736469a)